apcd_cms/src/client/src/components/Admin/Submissions/AdminSubmissions.tsx

```diff
@@ -66,9 +65,9 @@ export const AdminSubmissions: React.FC = () => {
     return (
       <SectionMessage type="error">
         There was an error loading the page.{' '}
-        <Link to="/workbench/dashboard/tickets/create" className="wb-link">
+        <a href="/workbench/dashboard/tickets/create" className="wb-link">
```

Comment:
Link does not work if the react app is not the same. BrowserRouter is not shared.